### PR TITLE
Add support for multiple files in fetch command

### PR DESCRIPTION
**Pull request by yamalight**
Hello!
I've tried addressing #3149 since it seemed like nobody else was working on it.
This PR adds basic support for multiple files (and globs expanded by shell), but doesn't address `"**.ts"` case (which as far as I understood from discussion wasn't needed)
My first time using Rust, so please let me know if there's anything I could've done better.

### CODE REVIEW

File: cli/flags.rs [hunk 1]
```diff
@@ -662,7 +664,12 @@ fn fetch_subcommand<'a, 'b>() -> App<'a, 'b> {
...
     .arg(no_remote_arg())
-    .arg(Arg::with_name("file").takes_value(true).required(true))
+    .arg(
+      Arg::with_name("file")
+        .takes_value(true)
+        .required(true)
+        .min_values(1),
+    )
     .about("Fetch the dependencies")
     .long_about(
```
Comment (ry) [inline cli/flags.rs]: At the bottom of this file there are some unit tests, please add one to demo the new CLI possibility.

File: cli/global_state.rs [hunk 2]
```diff
@@ -90,6 +90,15 @@ impl ThreadSafeGlobalState {
...
+      let files_len = flags.argv.len();
+      if files_len > 2 {
+        for i in 2..files_len {
+          let next_specifier = flags.argv[i].clone();
+          let _res = ModuleSpecifier::resolve_url_or_path(&next_specifier);
+        }
+      }
+      // then resolve root
       let root_specifier = flags.argv[1].clone();
       Some(ModuleSpecifier::resolve_url_or_path(&root_specifier)?)
```
Comment (ry) [inline cli/global_state.rs]: Does this change actually suffice to add support to multiple files in fetch? I'm a bit confused. I think a test would be useful see the `itest!` commands in `cli/tests/integration_test.rs`

File: cli/flags.rs (hunk 1 above)
Comment (yamalight) [reply cli/flags.rs]: Done!

File: cli/global_state.rs (hunk 2 above)
Comment (yamalight) [reply cli/global_state.rs]: That was indeed not enough. Fixed the logic, added integration test - but it currently fails.
Not entirely sure how to test this correctly as it seems like `Compile`/`Download` messages are being suppressed during tests - any hints?

Comment (bartlomieju) [reply cli/global_state.rs]: Add `check_stderr: true` to `itest!`

Comment (yamalight) [reply cli/global_state.rs]: That helped! Pushing the fix now, thanks!

File: cli/tests/integration_tests.rs [hunk 3]
```diff
@@ -320,6 +320,12 @@ itest!(_036_import_map_fetch {
   output: "036_import_map_fetch.out",
 });
 
+itest!(_037_fetch_multiple {
+  args: "fetch --reload fetch/test.ts fetch/other.ts",
+  check_stderr: true,
+  output: "037_fetch_multiple.out",
+});
+
```
Comment (bartlomieju) [inline cli/tests/integration_tests.rs]: Great, but unfortunately this test now relies on `unpkg.com` - could you change it to change import some other files present in `tests/` directory?

File: cli/tests/037_fetch_multiple.out [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Compile file:///home/yamalight/projects/bxjs/deno/cli/tests/fetch/test.ts
+Download https://unpkg.com/leaflet/dist/leaflet-src.esm.js
+Download https://unpkg.com/leaflet@1.6.0/dist/leaflet-src.esm.js
+Compile file:///home/yamalight/projects/bxjs/deno/cli/tests/fetch/other.ts
```
Comment (bartlomieju) [inline cli/tests/037_fetch_multiple.out]: These absolute paths will not work on CI, try this:
```
Compile file:///home/yamalight/projects/bxjs/deno/cli/tests/fetch/other.ts

Compile [WILDCARD]/fetch/other.ts
```

File: cli/tests/integration_tests.rs (hunk 3 above)
Comment (ry) [reply cli/tests/integration_tests.rs]: localhost:4545/cli/tests/hello.txt

File: cli/tests/037_fetch_multiple.out (hunk 4 above)
Comment (yamalight) [reply cli/tests/037_fetch_multiple.out]: how did I miss this 🤦‍♂ will fix in a second!

File: cli/tests/integration_tests.rs (hunk 3 above)
Comment (yamalight) [reply cli/tests/integration_tests.rs]: Sure, will do now!

Comment (yamalight) [reply cli/tests/integration_tests.rs]: Done!

File: cli/tests/037_fetch_multiple.out (hunk 4 above)
Comment (yamalight) [reply cli/tests/037_fetch_multiple.out]: And done!

File: cli/tests/integration_tests.rs [hunk 5]
```diff
@@ -320,6 +320,12 @@ itest!(_036_import_map_fetch {
   output: "036_import_map_fetch.out",
 });
 
+itest!(_037_fetch_multiple {
+  args: "fetch --reload fetch/test.ts fetch/other.ts",
+  check_stderr: true,
+  output: "037_fetch_multiple.out",
+});
```
Comment (bartlomieju) [inline cli/tests/integration_tests.rs]: Oooops, it needs explicit `http_server: true` as well, then it should work 👍

Comment (yamalight) [reply cli/tests/integration_tests.rs]: ah! just a second 😅

Comment (yamalight) [reply cli/tests/integration_tests.rs]: done!

**Comment by yamalight** [comment]
Awesome! 🥳
Thanks for all the guidance!

**Pull request merged**